packages/isomorphic/src/internal/normInput.ts

```diff
@@ -0,0 +1,11 @@
+import { Input } from './types'
+
+/**
+ * Normalize a string, number array, or Uint8Array to a string or Uint8Array.
+ * Both node and noble lib functions accept these types.
+ *
+ * @param input - value to normalize
+ */
+export default function normInput(input: Input): string | Uint8Array {
```

Comment:
```suggestion
export default function normalizeInput(input: Input): string | Uint8Array {
```
Easier to understand.